oam/install_scripts/mcs-ddlproc.service.in

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=mcs-ddlproc
-PartOf=mcs-exemgr.service
+PartOf=mcs-writeengineserver.service
```

Comment:
Modified and tested ordering change to remove a bug and error/warning from mcs-exemgr, mcs-ddlproc